packages/playground/website/src/components/modal/modal-buttons.tsx

```diff
@@ -0,0 +1,34 @@
+import { Button, Flex } from '@wordpress/components';
+import React from 'react';
+
+interface ModalButtonsProps {
+	submitText?: string;
+	areDisabled?: boolean;
+	areBusy?: boolean;
+	onCancel?: () => void;
+	onSubmit?: (e: any) => void;
+}
+export default function ModalButtons({ submitText = 'Submit', areDisabled = false, areBusy, onCancel, onSubmit }: ModalButtonsProps) {
+	return (
+		<Flex
+			justify={'end'}
```

Comment:
It's not a big deal, but if we're using a string, we can say just `justify="end"`, without the curly braces.

Reply:
I like visual consistency, so I pass strings in props so that they visually look like a variable. It's also easier to convert it to a variable or a string with a variable.

But that's my preference. I'll stick to your guidelinens!

Reply:
> I like visual consistency, so I pass strings in props so that they visually look like a variable.

Ah, I hadn't considered that. Thanks for sharing.